subcommands/common.go

```diff
@@ -132,6 +132,30 @@ func Tabby(indent int, columns ...interface{}) *tabby.Tabby {
 	return tab
 }
 
+func AssertWritable(path string) {
+	st, err := os.Stat(path)
+	DieNotNil(err)
+	f, err := os.OpenFile(path, os.O_APPEND|os.O_WRONLY, st.Mode())
```

Comment:
Interesting way to check whether a file is writable. I guess you checked `st.Mode()` approach.

Reply:
This is not mine invention, I just moved the @doanac 's code to a new place; and it has proven to work well for over a year.